src/i18n/en-US.json

```diff
@@ -1486,6 +1486,8 @@
   "replyBarCancelMessage": "Cancel answer",
   "replyBarEditMessage": "Edit",
   "replyBarLocation": "Edit",
+  "replyBarSingleAttachment": "+ 1 file",
```

Comment:
shouldn't this be +1 file? can we double check with Astrid please?

Reply:
probably it should, will change that